Tests/conf.json

```diff
@@ -5440,7 +5440,13 @@
                 "Demisto REST API"
             ],
             "playbookID": "FeedLOLBas_test"
-        }
+        },
+        [
+            {
+                "integrations": "CIRCL CVE Search",
+                "playbookID": "CIRCL CVE Search - Test"
+            }
+        ]
```

Comment:
Why is this definition within a list? remove if not required

Reply:
Because I tried doing 2 things at once :D